packages/@n8n/eslint-config/src/configs/base.ts

```diff
@@ -11,13 +11,6 @@ import tseslint from 'typescript-eslint';
 import eslintConfigPrettier from 'eslint-config-prettier/flat';
 import { createTypeScriptImportResolver } from 'eslint-import-resolver-typescript';
 
-// Slowest rules are disabled locally (for lint, not lintfix) to improve performance in development
-// They are enabled in CI to ensure code quality
-const runAllRules =
-	process.env.CI === 'true' ||
-	process.env.INCLUDE_SLOW_RULES === 'true' ||
-	process.argv.includes('--fix');
-
```

Comment:
This is intentional. Running `lintfix` locally and then checking the IDE which uses `lint` ends up showing in the IDE issues that will not show up in CI, because the `lint` ruleset is a subset of `lintfix` ruleset. The perf gain is not worth the confusion and inconvenience. To make linting more performant, we should consider switching to Oxlint or Biome instead.